sklearn/metrics/_regression.py

```diff
@@ -286,10 +286,15 @@ def mean_pinball_loss(
 def mean_absolute_percentage_error(
     y_true, y_pred, sample_weight=None, multioutput="uniform_average"
 ):
-    """Mean absolute percentage error regression loss.
+    """Mean absolute percentage error (MAPE) regression loss.
 
-    Note here that we do not represent the output as a percentage in range
-    [0, 100]. Instead, we represent it in range [0, 1/eps]. Read more in the
+        The MAPE computes the mean of the absolute value of the distance
+        between `y_true` and `y_pred` over `y_true`, which is a
+        relative error term.
+
+    Note here that the output is not a percentage in range [0, 100].
+    Instead, it can be arbitrarily high. Hence, the best value is 0
+    and the output is not upper bounded. Read more in the
```

Comment:
I think we can be more explicit here:

```
    Note here that the output is not a percentage in range [0, 100].
    Instead, the output can be arbitrarily high when `y_true` is small
    or when `abs(y_true-y_pred)` is large. Read more in the
```

(This is using your suggestion from the original issue.)